src/main/kotlin/net/atos/client/zgw/ztc/model/ZaaktypeInformatieobjecttypeListParameters.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.client.zgw.ztc.model
+
+import jakarta.ws.rs.QueryParam
+import java.net.URI
+
+class ZaaktypeInformatieobjecttypeListParameters(
+    private val zaaktype: URI
```

Comment:
I think you can use (similar as done on other fields) the annotation `@field:QueryParam("zaaktype")` directly in the constructor

Reply:
No, see above. That did not work.. Something to look at later maybe but when adding it to the constructor the query params were not added to the requests by JAX-RS..